Mathlib/LinearAlgebra/BilinearForm/TensorProduct.lean

```diff
@@ -63,29 +64,29 @@ theorem tensorDistrib_tmul (B₁ : BilinForm A M₁) (B₂ : BilinForm R M₂) (
     tensorDistrib R A (B₁ ⊗ₜ B₂) (m₁ ⊗ₜ m₂) (m₁' ⊗ₜ m₂')
       = B₂ m₂ m₂' • B₁ m₁ m₁' :=
   rfl
-#align bilin_form.tensor_distrib_tmul LinearMap.BilinForm.tensorDistrib_tmulₓ
+#align bilin_form.tensor_distrib_tmul LinearMap.BilinMap.tensorDistrib_tmulₓ
 
 /-- The tensor product of two bilinear forms, a shorthand for dot notation. -/
 @[reducible]
-protected def tmul (B₁ : BilinForm A M₁) (B₂ : BilinForm R M₂) : BilinForm A (M₁ ⊗[R] M₂) :=
```

Comment:
Should these be left as `BilinForm` now?

Reply:
I think I started moving some things into the `BilinMap` namespace to avoid breaking dot notation, but this then leaves a bit of a mishmash of `BilinForm`s and `BilinMap`s. And even with these changes I still ended up breaking dot notation elsewhere. Possibly we need to sort out `BilinForm` / `BilinMap` / (along with the Sesquilinear equivalents) before this PR?